src/main/sensors/compass.h

```diff
@@ -48,7 +49,8 @@ extern mag_t mag;
 typedef struct compassConfig_s {
     int16_t mag_declination;                // Get your magnetic decliniation from here : http://magnetic-declination.com/
                                             // For example, -6deg 37min, = -637 Japan, format is [sign]dddmm (degreesminutes) default is zero.
-    sensor_align_e mag_align;               // mag alignment
+    sensor_align_e mag_alignment;               // mag alignment
```

Comment:
Not your code, but the type of this should be changed to `uint8_t` to give it a fixed size when persisted.

Reply:
That will break the structure :-(
Maybe it's time to increase version ...

Reply:
Added a separate commit that does both of the things you each suggested as well as some other alignment/ordering tweaks.